CHANGELOG.md

```diff
@@ -36,8 +42,6 @@
 ### Features
 
 * **Functionality to catch and classify overlapping/nested elements** Method to identify overlapping-bboxes cases within detected elements in a document. It returns two values: a boolean defining if there are overlapping elements present, and a list reporting them with relevant metadata. The output includes information about the `overlapping_elements`, `overlapping_case`, `overlapping_percentage`, `largest_ngram_percentage`, `overlap_percentage_total`, `max_area`, `min_area`, and `total_area`.
-* **Add Local connector source metadata** python's os module used to pull stats from local file when processing via the local connector and populates fields such as last modified time, created time.
-* **Add Local connector source metadata.** python's os module used to pull stats from local file when processing via the local connector and populates fields such as last modified time, created time.
```

Comment:
mistaken deletion?

Reply:
Oh, not exactly. 
That line (that was duplicated indeed) was moved to 
https://github.com/Unstructured-IO/unstructured/blob/412c26067614454f759bb37a83f9f4348ded1ec8/CHANGELOG.md?plain=1#L12
The changelog got old during development, so it should be in most recent -dev version